backend/database/actionsWithTables/categoryActions.js

```diff
@@ -0,0 +1,65 @@
+const { Category } = require('../tables/category');
+const { Subcategory } = require('../tables/subcategory');
+
+const addNewCategory = async (data) => {
+    return await Category.create({
+        name: data.name,
+    });
+};
+
+const getAllCategories = async () => {
+    const categories = await Category.findAll({
+        attributes: ['category_id', 'name', 'is_deleted'],
+        where: {
+            is_deleted: false,
+        },
+    });
+    return await categories;
+};
+
+const getCategory = async (id) => {
+    const category = await Category.findAll({
+        attributes: ['category_id', 'name', 'is_deleted'],
+        where: {
+            category_id: id,
+            is_deleted: false,
```

Comment:
Remove `is_deleted` in `getCategory(id)`. It must return entity even if it deleted - `is_deleted` for list filtering, not ID